apps/frontend/src/pages/access-management/libs/components/group-users-table/group-users-table.tsx

```diff
@@ -85,7 +88,11 @@ const GroupUsersTable = ({
 		<>
 			<div>
 				<span className={styles["table-title"]}>Users</span>
-				<GroupUsersSearch onChange={handleSearchChange} />
+				<GroupUsersSearch
```

Comment:
looks like we need to pass name to the search too, because inside it we don't know how we named it in form values